src/App.php

```diff
@@ -58,6 +58,10 @@
  */
 class App
 {
+	const PLATFORM = 'Friendica';
```

Comment:
@nupplaphil does this change affect the CI/build scripts somehow?

Reply:
No, I need the dbstructure.conf.php and the dbstructure.sql Version values, but not this one :-)